.travis.yml

```diff
@@ -1,12 +1,12 @@
-dist: trusty
+dist: xenial
```

Comment:
is there appveyor setup for this repo? the same changes expected to happen there

Reply:
Nope. That's a battle for another time.